libs/designsystem/item/src/item.component.spec.ts

```diff
@@ -113,44 +113,52 @@ describe('ItemComponent', () => {
         spectator.setInput('rotateIcon', true);
       });
 
-      it('should rotate the icon 180deg if the disclosure icon is "arrow-down', async () => {
+      it('should rotate the icon 180deg if the disclosure icon is "arrow-down"', async () => {
         spectator.setInput('disclosure', 'arrow-down');
+        spectator.detectChanges();
 
         const icon = spectator.query<HTMLElement>('kirby-icon');
-        await TestHelper.waitForTimeout(); // Wait a tick
+        const ionIcon = spectator.query<HTMLElement>('ion-icon');
+        await TestHelper.ionComponentOnReady(ionIcon);
```

Comment:
Since this is a bit of a roundabout way of ensuring that kirby-icon has been rendered, consider adding a comment to the relevant tests explaining this .